.github/workflows/deploy-main-job-function.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
 
       - name: Install dependencies
-        run: npm ci
```

Comment:
This was on purpose, as digital ocean does not want the package lock json file to be included and then you need to run the full install